sources/packages/forms/src/form-definitions/sfaa2122.json

```diff
@@ -2337,7 +2337,7 @@
       "tableView": false,
       "components": [
         {
-          "html": "<p>For the purpose of verifying the data provided in this application for student assistance, I hereby consent to the release, by the Canada Revenue Agency, to the Ministry of Advanced Education, Skills and Training (or a person delegated by the ministry), of taxpayer information from any portion of my income tax records that pertains to information given by me on any StudentAid BC application.</p>\n\n<p>The information will be relevant to, and used solely for the purpose of determining and verifying my information and for my spouse&rsquo;s eligibility for and entitlement to the following programs: Canada Student Grant for students with Permanent Disabilities, the BC Supplemental Bursary for Students with a Permanent Disability, Canada Student Grant for Part-time Students, Canada Student Loan for Part-time Studies, and if eligible, Canada Student Grant for Part-time Students with Dependants under the Canada Student Financial Assistance Act.</p>\n\n<p>This information will not be disclosed to any other person or organization without my prior approval. This authorization is valid for the two taxation years prior to the year of signature of this consent, the year of signature of this consent and for any other subsequent consecutive taxation year for which assistance is requested.</p>\n",
+          "html": "<p>For the purpose of verifying the data provided in this application for student assistance, I hereby consent to the release, by the Canada Revenue Agency, to the Ministry of Post Secondary and Future Skills (or a person delegated by the ministry), of taxpayer information from any portion of my income tax records that pertains to information given by me on any StudentAid BC application.</p>\n\n<p>The information will be relevant to, and used solely for the purpose of determining and verifying my information and for my spouse&rsquo;s eligibility for and entitlement to the following programs: Canada Student Grant for students with Permanent Disabilities, the BC Supplemental Bursary for Students with a Permanent Disability, Canada Student Grant for Part-time Students, Canada Student Loan for Part-time Studies, and if eligible, Canada Student Grant for Part-time Students with Dependants under the Canada Student Financial Assistance Act.</p>\n\n<p>This information will not be disclosed to any other person or organization without my prior approval. This authorization is valid for the two taxation years prior to the year of signature of this consent, the year of signature of this consent and for any other subsequent consecutive taxation year for which assistance is requested.</p>\n",

```

Comment:
This file was not supposed to be on the repo anymore. @dheepak-aot did we not remove all the non-used forms definitions? I remember we talked about it but I don't know if we did it or just created a tech debt.
@sh16011993 the file can be removed, IMO.

Reply:
Removed the file.

Reply:
Yes @andrewsignori-aot . Technical debt https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/1705 was created to remove obsolete forms. But we did not do this ticket yet. 
@sh16011993 If this file is removed from her you can remove it's name from the above mentioned ticket.